spack/package.py

```diff
@@ -155,12 +162,9 @@ def url_for_version(self, version):
     conflicts("+netcdf3", when="+netcdf")
     conflicts("+openmp", when="%apple-clang")
 
-    depends_on("wget", type=("build"), when="@:3.1 +netcdf4")
-    depends_on("wget", type=("build"), when="@:3.1 +netcdf")
     depends_on("ip@5.1:", when="@develop +ipolates")
     depends_on("libaec@1.0.6:", when="@3.2: +aec")
     depends_on("netcdf-c", when="@3.2: +netcdf4")
-    depends_on("netcdf-c", when="@3.2: +netcdf")
     depends_on("jasper@:2", when="@3.2: +jasper")
     depends_on("zlib-api", when="+png")
```

Comment:
I think these three lines should also be "@3.2:" since the old wgrib2 doesn't have any external deps (apart from 'make')?